docker/linux-runner

```diff
@@ -140,12 +143,16 @@ esac
     else
         ${QEMU_CMD} >> "${LOG}" 2>&1 &
     fi
+    qemu_pid=$!
 
     # wait for dropbear
     for _ in $(seq 240); do
         if grep -q "Not backgrounding" "${LOG}"; then
             READY=1
             break
+        elif ! (ps -p "${qemu_pid}" >/dev/null 2>&1); then
```

Comment:
This fixes an issue where if `QEMU_CMD` fails early for any reason, then we will attempt to boot via dropbear for at least 120s. With checking if the process ID still exists for the Qemu command, we can exit early if the command fails.